lib/widgets/avatar_view/avatar_view.dart

```diff
@@ -49,8 +50,13 @@ class ConversationAvatarWidget extends HookWidget {
             if (_category == ConversationCategory.group) {
               return context.database.participantDao
                   .participantsAvatar(_conversationId!)
-                  .watchThrottle(const Duration(minutes: 6))
-                  .map((event) => _category == ConversationCategory.contact
+                  .watchWithStream(
```

Comment:
group avatar widget should has a leading event, otherwise there is no icon when widget first rendering....

Reply:
Cause watchWithStream.prepend default to true, that will not happen.